common/src/main/java/com/wynntils/features/user/inventory/InventoryEmeraldCountFeature.java

```diff
@@ -47,8 +47,7 @@ public class InventoryEmeraldCountFeature extends UserFeature {
 
     @SubscribeEvent
     public void onContainerRender(ContainerRenderEvent event) {
-        int emeralds =
-                ContainerUtils.getEmeraldCountInContainer(event.getScreen().getMenu());
+        int emeralds = ContainerUtils.getEmeraldCountInContainer(McUtils.containerMenu());
```

Comment:
What has changed?

Reply:
This is more consistent with the way I calculate the inv contents. It is the same thing basically though.

Reply:
ah